src/pmdas/bpf/modules/netatop.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+ * Copyright (c) 2021 Bytedance
+ * https://github.com/bytedance/netatop-bpf
+ */
+
+#ifndef __NETATOP__
+#define __NETATOP__
+struct taskcount {
+	unsigned long long	tcpsndpacks;
+	unsigned long long	tcpsndbytes;
+	unsigned long long	tcprcvpacks;
+	unsigned long long	tcprcvbytes;
+
+	unsigned long long	udpsndpacks;
+	unsigned long long	udpsndbytes;
+	unsigned long long	udprcvpacks;
+	unsigned long long	udprcvbytes;
+
+	/* space for future extensions */
+};
+#define	COMLEN	16
+
+// struct netpertask {
+// 	char type; 	// tgid or tid or
+// 	pid_t			id;	// tgid or tid (depending on command)
+// 	struct taskcount	tc;
+// };
+
+struct netpertask {
+	pid_t			id;	// tgid or tid (depending on command)
+	unsigned long		btime;
+	char			command[COMLEN];
+
+	struct taskcount	tc;
+};
+
+extern struct netatop_bpf *skel;
+extern int semid;
+extern int tgid_map_fd;
+extern int tid_map_fd;
+extern int nr_cpus;
+
+#define	NUMCLIENTS	(semctl(semid, 1, GETVAL, 0))
+#define SEMAKEY         1541962
```

Comment:
Are NUMCLIENTS and SEMAKEY used?  I may be missing it, but if not let's drop these two lines.